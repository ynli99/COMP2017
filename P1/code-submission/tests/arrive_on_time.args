London Paris "15:30:10"
