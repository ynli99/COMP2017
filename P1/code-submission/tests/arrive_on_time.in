London::Paris::10:00:30
London::Paris::15:30:10
Paris::London::14:50:00
Paris::London::20:40:50
