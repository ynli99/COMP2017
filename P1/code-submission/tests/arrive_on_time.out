The next train to Paris from London departs at 15:30:10
